visidata/_types.py

```diff
@@ -53,7 +54,7 @@ def getType(typetype):
 vdtype(anytype, '', formatter=lambda _,v: str(v))
 vdtype(str, '~', formatter=lambda _,v: v)
 vdtype(int, '#', '%.0f')
-vdtype(float, '%', '%.02f')
+vdtype(float, "%", "%.0{}f".format(options.disp_float_ndigits))
```

Comment:
Using `options` at the toplevel like this only uses the default, because it's evaluated before options parsing (including .visidatarc) happens.  Also it looks like `vtask` is just resetting the default.  Do you have a use for this configurability yourself?